docs/security/host-certs.md

```diff
@@ -27,25 +33,28 @@ The OSG PKI Command Line Clients are tested to work on Python version 2.4+. They
 -   the name of the **Virtual Organization** affiliated with the Certificate Authority
 
 Requesting host/service certificate using OIM
-==================================================
+----------------------------------------------
 
-The OSG PKI Certificate Request & Management System can be found at: <https://oim.opensciencegrid.org/oim/certificate>.
+The OSG PKI Certificate Request & Management System can be found at:
+<https://oim.opensciencegrid.org/oim/certificate>.
 
-For instructions on how to request a host or service certificate using the Web interface please see the [user guide maintained by GOC](https://confluence.grid.iu.edu/pages/viewpage.action?pageId=3244064).
+For instructions on how to request a host or service certificate using the Web
+interface please see the [user guide maintained by
+GOC](https://confluence.grid.iu.edu/pages/viewpage.action?pageId=3244064).
 
 Installation
-=================
+-------------
 
-Install the OSG PKI Command Line Clients 
-----------------------------------------------------------------------------------
+### Install the OSG PKI Command Line Clients 
 
 ``` console
-[root@host /]$ yum install osg-pki-tools
+root@host # yum install osg-pki-tools
 ```
 
-### Configuration file pki-clients.ini
+#### Configuration file pki-clients.ini
 
-This configuration is correctly set by default and you will not need to change them. The information is provided in case you need it for debugging purposes.
+This configuration is correctly set by default and you will not need to change
+them. The information is provided in case you need it for debugging purposes.
```

Comment:
If this is rarely-if-ever needed, bump it down to an appendix where it will get rarely-if-ever read.

Reply:
Moved this to the end of the FAQ.  However, is this information really needed?  Seems like anyone that needs it will already know about it (i.e. they are developers).